docs/_partners/planning-guide.md

```diff
@@ -118,7 +118,8 @@ Each time, users expect consistent messaging and with the items they'd expect to
 
 ## Authentication
 
-![Example OAuth v2 Authentication Screen](https://cdn.zapier.com/storage/photos/6f14340218bd51f4047afcdfc71fd9b6.png)
+![Example OAuth v2 Authentication Screen](https://cdn.zappy.app/7755d2ed759d2db71a65b121f31026ca.png)
```

Comment:
FYI: this screenshot doesn't blur out the last name in the email address, pointing that out since the last name was blurred

Reply:
@Jesswinters thanks for the catch, I did modify it, if you have a chance to approve again please and I'll merge.